CustomDimensions.php

```diff
@@ -280,10 +281,13 @@ public function addConversionInformation(&$conversion, $visitInformation, Tracke
 
         foreach ($dimensions as $dimension) {
             $index = (int) $dimension['index'];
-            if ($dimension['scope'] === self::SCOPE_VISIT && in_array($index, $conversionIndexes)) {
+            if (in_array($index, $conversionIndexes)) {
                 $field = LogTable::buildCustomDimensionColumnName($dimension);
 
-                if (array_key_exists($field, $visitInformation)) {
+                if ($dimension['scope'] === self::SCOPE_ACTION && array_key_exists($field, $actionDimensions)) {
+                    $conversion[$field] = $actionDimensions[$field];
+                }
+                else if ($dimension['scope'] === self::SCOPE_VISIT  && array_key_exists($field, $visitInformation)) {
```

Comment:
Thanks @tsteur for your comments, I understand, and agree that this requires a more complex solution. Last week I also placed a comment under the original issue, that there is a problem fetching report data with log_conversion custom dimensions (couldn't get it to work). I am starting to think that maybe for now I should just try to fix it the other way around: creating an action also for manual conversions. Then everything else should work automatically. But as always, suggestions are welcome.